### ISSUE TITLE
Ask WMTS directly to the server

### ISSUE THREAD

**Issue by katagen** [3liz/lizmap-web-client#4158]
### What is the question?

In the plugin, we can ask that the layer, if it is a WMS, to be served directly from the server.
Would it be possible also for WMTS?

### Versions, safeguards, check summary etc

Last LTR of all.

### Check Lizmap plugin

- [ ] I have done the step just before in the Lizmap QGIS desktop plugin before opening this ticket. Otherwise, my ticket is not considered valid and might get closed.

### Operating system

Ubuntu

### Browsers

Chrome

### Browsers version

Last

**Comment by Gustry** [comment]
The "WMTS" script available in the JS library ? https://github.com/3liz/lizmap-javascript-scripts/blob/master/README.md

**Comment by mdouchin** [comment]
Unfortunately, this JS script will not work for Lizmap Web Client >= 3.7 .

**Comment by katagen** [comment]
No, in the plugin, if I have WMS layer in my project, I can ask LWC to load the data directly from the original server. My question is why it is not possible to do so for WMTS?

**Comment by katagen** [comment]
![lwc](https://github.com/3liz/lizmap-web-client/assets/37768582/96c15400-407d-4a49-a4dc-cba5f1edc0b3)

**Comment by katagen** [comment]
Is the question clear enough? Or does it need more feedback?

**Comment by Gustry** [comment]
Did you check the purpose of the script provided above ?

**Comment by katagen** [comment]
Just under your comment, @mdouchin said that it will not work with LWC 3.7

**Comment by mdouchin** [comment]
Lizmap Web Client does not support the feature related to the checkbox "Get images directly from the ..." for WMTS layers. The support is only for WMS layers. 

If we want to implement this feature also for WMTS layers, it means LWC must request the GetCapabilities from the external server, do some checks, and then add the layers with reprojection (if needed). It would be great but not supported yet.

Could you help us fund this feature ?

**Comment by katagen** [comment]
I can ask... but I probably need precision about what that means!

I'm sure you know what and why 😉

**Comment by nboisteault** [comment]
WMTS layers requests are made directly to the server when they are in the `baselayers` group. Unfortunately it only works when the projection is 3857. A development has to be made but it needs funding.
